10_K_Risk_Evaluator/path_mover.py

```diff
@@ -0,0 +1,73 @@
+#Objective: Navigate file paths to access html documents prior to feeding into html parser
+from html_parser import import_data
+from bs4 import BeautifulSoup, NavigableString, Tag
+from pathlib import Path
+import os
+
+
+cwd = Path('.')
+#local location that will sync with where user downloads data
+data = Path('/Users/nick/Documents/cs506/project')
+
+# full list of directory's with html files
+#531 total
+full_set = list(data.glob('**/*.html'))
+first_10 = full_set[:10]
+file = first_10[0]
+#attempting to link path to Eric's import data functions
+#testing each function
+test = import_data(str(first_10[0]))  # creates populated soup object based on file path
+
+
+
+
+
+
+
+
+
+
+#terms from grab section
+start_terms = ['ITEM', '1A.', 'RISK', 'FACTORS']
+end_terms = ['ITEM', '1B.', 'UNRESOLVED', 'STAFF', 'COMMENTS']
+
+def grab_section_text(soup_object, start_terms, end_terms):
```

Comment:
So we've made a copy of grab_section_text() from html_parser.py?
Will this code eventually be part of hml_parser, or remain a stand-alone module?

Reply:
No it should be part of the html parser. I made a copy because i was able to feed in a file path and create a populated soup object. ( so your import_data function worked with the new file paths. The grab_section function kept returning blank lists when i fed it the populated soup object so i was trying to debug it function by function.  Let me know if this makes sense or not.

It terms of where this code will go, im not sure i want it to get it to feed into your main function first. It may be easier to keep it separate for now just till we get it working how we want.